framework/src/base/MooseApp.C

```diff
@@ -65,7 +65,9 @@ validParams<MooseApp>()
       "display_version", "-v --version", false, "Print application version");
   params.addCommandLineParam<std::string>("input_file", "-i <input_file>", "Specify an input file");
   params.addCommandLineParam<std::string>(
-      "mesh_only", "--mesh-only", "Setup and Output the input mesh only.");
+      "mesh_only",
+      "--mesh-only [mesh_file_name]",
+      "Setup and Output the input mesh only (Default: \"input_file_in.e\")");
```

Comment:
How about `<input_file_base>_in.e`?

Reply:
Hmm, ok my default isn't quite good enough, but "input_file_base" doesn't seem quite right. I might change it a little. Let me see if the core dump can be fixed easily.